express-101/src/index.js

```diff
@@ -1,9 +1,11 @@
 const express = require('express');
 
 const app = express();
-
-// 1) Create Routes
-
-// 2) Start server on port 3000
-
+app.get('/', (req, res) => {
+  res.send('Hey, I am server response');
```

Comment:
Should you not use `return` statement here?
Like `return res.send();`
It actually make sure, that you are not doing anything else by mistake after sending response.
Here is one stackoverflow discussion on same
https://stackoverflow.com/questions/43055600/app-get-is-there-any-difference-between-res-send-vs-return-res-send